.travis.yml: set go_import_path

srenatus opened the pull request:
When pushing a PR to _my_ remote (github.com/ME), travis will send me an
email saying that it failed to build, and keeps failing, for every
update of [the branch](https://travis-ci.com/srenatus/opa/builds/86468169).

Setting this `go_import_path` properly should fix the issue.

Update: [✅ looks good from here](https://travis-ci.com/srenatus/opa/builds/86473448)

srenatus commented:
Nah, this stops somewhere further down now, [here](https://travis-ci.com/srenatus/opa/builds/86473448#L662)

tsandall commented:
The CI is set up to push the images to the openpolicyagent repository on Docker Hub. What should the behaviour be in this case?

srenatus commented:
Updated the PR with adding a check for another travis' env that should differentiate fork vs pull requests vs merge to `github.com/open-policy-agent/opa` master.

⚠️ Naturally couldn't test that this still works as expected _in this repo_; so, if we merge this, a close watch would be better. 👀

srenatus commented:
:x: doesn't work as expected. Will revisit.

srenatus commented:
⏩ Tries switching to Travis' [deployments feature](https://docs.travis-ci.com/user/deployment#supported-providers). It provides a nice declarative way for saying "only do this on open-policy-agent/opa" and "only for tags". Not sure if works just like that, though; the caveat from above still applies.

tsandall commented on .travis.yml:
```diff
@@ -9,9 +10,16 @@ env:
 services:
 - docker
 after_success:
```
Can we just delete `after_success` now?

srenatus commented:
I've cleaned up the empty key and squashed the commits.

> The only thing I was not sure about was whether the second deploy item will run after the first deploy item (because the release-travis target does not build the image, which is required for the tag-latest/push-latest targets.)

I am not sure, and [the docs](https://docs.travis-ci.com/user/deployment#deploying-to-multiple-providers) are not specific about that. Let's give it a try, I guess?

srenatus commented:
ℹ️ looks like it was skipped [in the same order here](https://travis-ci.com/srenatus/opa/jobs/149393570#L646), for what it's worth 🤔

tsandall commented:
OK, let's give it a shot.
